src/Adliance.Buddy.CodesStyle/Adliance.Buddy.CodesStyle.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <Version>0.0.0</Version>
+    <Authors>Alexander Freistetter</Authors>
+    <Company>Adliance GmbH</Company>
+    <Product />
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Copyright>(c) Adliance GmbH</Copyright>
+    <PackageProjectUrl>https://www.adliance.net</PackageProjectUrl>
+    <Description>Common .editorconfig file.</Description>
+    <Nullable>enable</Nullable>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <NoDefaultExcludes>true</NoDefaultExcludes><!-- include files starting with . -->
+    <PackageReadmeFile>readme.md</PackageReadmeFile>
+  </PropertyGroup>
+  
+  <!-- Select the MSBuild props and composed .editorconfig files to pack -->
+  <ItemGroup>
+    <None Include="build\Adliance.Buddy.CodesStyle.props" Pack="true" PackagePath="build\" />
+    <None Include="Rules\.editorconfig" Pack="true" PackagePath="content\rules"></None>
+    <None Include="readme.md" Pack="true" PackagePath="\"/>
```

Comment:
Why include the `readme` in the package?

Reply:
To render the readme on nuget.org:
https://devblogs.microsoft.com/nuget/add-a-readme-to-your-nuget-package/

Do you know a different way?

Reply:
No ;) Did not think about that, thanks.